kraken-app/kraken-app-portal/src/__mocks__/users.json

```diff
@@ -0,0 +1,326 @@
+[
+  {
+    "id": "33f9ff66-45d4-4b91-9a80-e24c1ca8f468",
+    "createdAt": "2024-10-15T07:09:58.172886Z",
+    "createdBy": "3f18feea-099c-4ff6-bb6d-9159ff68c8f3",
+    "name": "Qkuinzy Guingab",
+    "email": "qguingab@gg.cc",
+    "role": "ADMIN",
+    "state": "ENABLED",
+    "productId": "",
+    "updatedAt": "",
+    "updatedBy": ""
+  },
+  {
+    "id": "5f88aae9-9770-4648-833c-3df61f45ef27",
+    "createdAt": "2024-10-15T07:09:32.944189Z",
+    "createdBy": "3f18feea-099c-4ff6-bb6d-9159ff68c8f3",
+    "name": "Terrence Duhaylungsod",
+    "email": "duhaylungso@gg.cc",
+    "role": "ADMIN",
+    "state": "ENABLED",
+    "productId": "",
+    "updatedAt": "",
+    "updatedBy": ""
+  },
+  {
+    "id": "1c162277-65d2-40e7-80d9-5e88a5c4eeae",
+    "createdAt": "2024-10-15T07:09:09.726006Z",
+    "createdBy": "3f18feea-099c-4ff6-bb6d-9159ff68c8f3",
+    "name": "Francis Pagaragan",
+    "email": "fpagaragan@gg.cc",
+    "role": "ADMIN",
+    "state": "ENABLED",
+    "productId": "",
+    "updatedAt": "",
+    "updatedBy": ""
+  },
+  {
+    "id": "97bd2a0c-6cd9-44ca-b3a3-50920ed4fb3d",
+    "createdAt": "2024-10-15T07:08:11.053317Z",
+    "createdBy": "3f18feea-099c-4ff6-bb6d-9159ff68c8f3",
+    "name": "Argel Castillo",
+    "email": "acastillo@gg.cc",
+    "role": "ADMIN",
+    "state": "ENABLED",
+    "productId": "",
+    "updatedAt": "",
+    "updatedBy": ""
+  },
+  {
+    "id": "e5543373-8e30-4e00-b6d1-cf95495de9c7",
+    "createdAt": "2024-10-14T02:11:43.620024Z",
+    "createdBy": "3050ef61-3dc4-4732-9cef-dc316c48beef",
+    "name": "james",
+    "email": "tnhuy@cmcglobal.vn",
```

Comment:
please use a mock email here, do not appear any third party company information.